src/Components/Item/Item.js

```diff
@@ -0,0 +1,31 @@
+import "./Item.css";
+import { Link } from "react-router-dom";
+import { useState, useEffect } from "react";
+
+const Item = ({ icon, itemName = null }) => {
+  const [isExpanded, setisExpanded] = useState(!itemName);
```

Comment:
The setter for this should be setIsExpanded.
If we are using is in state, it means that it must be a boolean. Therefore, it should not have a default value of !itemName as itemName will either be null or some string.